### PR TITLE
Enabled dashboard plugin - Partially closes 1528 in lisk-docs

### LINKED ISSUE #1528: Update `Using dashboard with the blockchain application`
### Description

* Update `Using dashboard with the blockchain application` to use v6 interfaces
* Update terminologies to Lisk SDK v6
* Update images using Lisk SDK v6

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Partially closes [1528](https://github.com/LiskHQ/lisk-docs/issues/1528) in lisk-docs

- Bump up versions of node & packages, including `lisk-sdk: beta 4`.
- Enable the `dashboard` plugin.
- Enable the `allowedMethods` property in custom config.

### CODE REVIEW

File: tutorials/hello/hello_client/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	"homepage": "",
 	"repository": {},
 	"engines": {
-		"node": ">=16.14.1 <=16",
+		"node": ">=18.12.0 <=18",
```
Comment (Tschakki) [inline tutorials/hello/hello_client/package.json:17]: I get error `dyld[44834]: missing symbol called` when grying to start the client. 
I run node v18.17.0, not sure if it is related.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: tutorials/hello/hello_client/package.json (hunk 1 above)
Comment (TalhaMaliktz) [reply tutorials/hello/hello_client/package.json:17]: No error for me! Can you please delete `node_modules` and the `package-lock.json` file and do `npm i` again?

Comment (Tschakki) [reply tutorials/hello/hello_client/package.json:17]: It was a node.js error, this solved it: https://gist.github.com/adrienjoly/e29a6e73fb7e701eefd80ff9bde9abeb

**Review by Tschakki — APPROVED** [review]
LGTM 👍

**Review by mitsuaki-u — APPROVED** [review]

**Pull request merged**